.gitignore

```diff
@@ -2,6 +2,7 @@ npm-debug.log*
 node_modules/
 
 build/
+coverage/
 *.tgz
 
 .DS_Store
```

Comment:
might want to move this to a more standard `.gitignore` I use for JS stuff nowadays (`react-signature-canvas` should also be updated to that), though this is working fine for now